<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrm</class>
 <widget class="QDialog" name="MainFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleTimer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/SimpleTimerLogo.png</normaloff>:/SimpleTimerLogo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QSpinBox" name="minSelect">
         <property name="whatsThis">
          <string>Specify how many minutes to count down from here.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="showWarnChk">
         <property name="whatsThis">
          <string>If this is checked when &quot;Start&quot; is pressed, the &quot;5 minutes left&quot; and &quot;30 seconds left&quot; warnings will not be shown.</string>
         </property>
         <property name="text">
          <string>Don't show warnings</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>27</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="startBtn">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>27</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="aboutSTBtn">
            <property name="whatsThis">
             <string>This button shows SimpleTimer's &quot;About&quot; dialog.</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/SimpleTimerLogo.png</normaloff>:/SimpleTimerLogo.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="aboutQtBtn">
            <property name="whatsThis">
             <string>This button shows Qt's &quot;About&quot; dialog.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Qt.png</normaloff>:/Qt.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timePage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="timeLbl">
         <property name="whatsThis">
          <string>This label tells you how much time is left.</string>
         </property>
         <property name="text">
          <string>00 : 00</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pauseBtn">
         <property name="whatsThis">
          <string>Pause/resume the timer.</string>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="resetBtn">
         <property name="whatsThis">
          <string>Pressing this button will reset the time left to the original value.</string>
         </property>
         <property name="text">
          <string>Reset Time</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
